core/variant.cpp

```diff
@@ -3519,3 +3519,5 @@ String vformat(const String &p_text, const Variant &p1, const Variant &p2, const
 
 	return fmt;
 }
+
+void collect_args(const Array &args){};
```

Comment:
Why does this need to be here? It's defined and implemented in the header already.